azurerm/internal/services/machinelearning/machine_learning_inference_cluster_resource.go

```diff
@@ -92,22 +92,39 @@ func resourceAksInferenceCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"cert": {
-							Type:     schema.TypeString,
-							Optional: true,
-							ForceNew: true,
-							Default:  "",
+							Type:          schema.TypeString,
+							Optional:      true,
+							ForceNew:      true,
+							Default:       "",
+							ConflictsWith: []string{"leaf_domain_label", "overwrite_existing_domain"},
```

Comment:
Persumably should be:

```suggestion
							ConflictsWith: []string{"ssl.0.leaf_domain_label", "ssl.0.overwrite_existing_domain"},
```

Reply:
@magodo Yes you are right, but I do not understand why as the parameters are defined within the same block.

Reply:
@gro1m Unfortunately this is how the resource data is implemented internally, as a map keyed by property addresses.